src/domain/enrolment/EditEnrolmentPage.tsx

```diff
@@ -145,10 +176,18 @@ const EditorEnrolmentPage: React.FC = () => {
                 {t('enrolment.editEnrolmentBackButton')}
               </BackButton>
               <h1>{t('enrolment.editEnrolmentTitle')}</h1>
-              <EnrolmentForm
-                onSubmit={handleSubmit}
-                initialValues={initialValues}
-              />
+              {!!minGroupSize && !!maxGroupSize ? (
+                <EnrolmentForm
+                  onSubmit={handleSubmit}
+                  initialValues={initialValues}
+                  minGroupSize={minGroupSize}
+                  maxGroupSize={maxGroupSize}
+                />
+              ) : (
+                <Notification label="Virhe" type="error">
+                  Virhe lomakkeen alustuksessa
+                </Notification>
```

Comment:
I will ask for translatio if we are going to keep this here. Not 100% sure how we shoul to this kind of error handling, shouldn't if nothing strange happens in API

Reply:
It's good to inform that something is not ok.